http/http-sse/src/test/java/http/sse/HttpSseIT.java

```diff
@@ -0,0 +1,27 @@
+package http.sse;
+
+import static io.restassured.RestAssured.given;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+
+import io.quarkus.test.scenarios.QuarkusScenario;
+
+@QuarkusScenario
+@Tag("QQE-257")
```

Comment:
This is very new thing, I'm not sure it is necessary, but it really depends on team discussion :-/ We are using tags for Quarkus tickets, but there is a difference, because this is a ticket used for work planning.

Reply:
My POV on this is that it's ok use tag but I would change qqe to quarkus issue. It make more sense for me, also qqe is only internal.

Reply:
Alright, thanks.

Reply:
OK I will change this. Is it better to have it without tag, or can I add a link to the upstream issue to the tag? AFAIK there is no quarkus jira for this.

Reply:
@mocenas this was more of a question, I really don't know. We can ask for @rsvoboda opinion, but as far as I'm concerned, there is nothing wrong on your approach. it just surprised me :-)

Reply:
> AFAIK there is no quarkus jira for this.

indeed